content/source/docs/cloud/sentinel/import/tfconfig-v2.html.md

```diff
@@ -177,7 +177,7 @@ resource value, one could do:
 import "tfconfig/v2" as tfconfig
 
 main = rule {
-	tfconfig.outputs["instance_id"].expressions.references is ["aws_instance.foo"]
+	tfconfig.outputs["instance_id"].value.references is ["aws_instance.foo"]
```

Comment:
Just basing off PR desc -- looks like `expressions` is supposed to be `config` here

Reply:
Oh nvm, this is outputs, right?  So this looks correct I think?

Reply:
Yeah this is good as per the latest PR to change this to `value` in the API.